plugins/filter_kubernetes/kube_conf.c

```diff
@@ -61,6 +62,15 @@ struct flb_kube *flb_kube_conf_create(struct flb_filter_instance *ins,
         return NULL;
     }
 
+    /* Token Command */
+    cmd = flb_filter_get_property("token_command", ins);
```

Comment:
As noted in Drew's comment, this option should be mutually exclusive with the existing token file option.